### PR TITLE
chore(kommander): bump chart reference to v0.4.4

### PR DESCRIPTION
**Pull request by jimmidyson**
**What type of PR is this?**
Chore

**What this PR does/ why we need it**:
Upgrades kommander addon to latest 0.4.0 release

**Which issue(s) this PR fixes**:
No issue

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: addons/kommander/1.x/kommander-5.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-4"
```
Comment (juliangieseke) [inline addons/kommander/1.x/kommander-5.yaml]:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-5"
```

Comment (jimmidyson) [reply addons/kommander/1.x/kommander-5.yaml]: Oh man I hate this...

**Comment by jaxesn** [comment]
I think I understand and accept why we are doing this, but I'm not sure I love this workflow... we are updating this addon to make it easier to do some integration testing with the UI team? Once this in master, since we cut stable tags off of master that means this would be included.... This is probably fine, but feels more risky than it needs to be.  Should we be using a beta branch?  Should we not merge this and just test off this branch?  Should there be other ways to test this out and only merge the addon bump once kommander, the Product (ui + kcl), is ready? cc @joejulian

**Comment by jimmidyson** [comment]
@jaxesn Yes the plan is to test with this branch as discussed in slack.

**Comment by jaxesn** [comment]
Ok that makes me feel better than.

**Comment by jaxesn** [comment]
There is no rush on this at this time.  Thanks for the rebase tho, I will test this with the latest dex and what not.

**Comment by jimmidyson** [comment]
PR is still a WIP but updated to latest kommander chart v0.4.3.

**Pull request closed without merging**